packages/orm/src/collections/CustomReference.ts

```diff
@@ -110,13 +97,23 @@ export class CustomReference<T extends Entity, U extends Entity, N extends never
   onEntityDelete(): void {}
   async refreshIfLoaded(): Promise<void> {}
 
+  /** Finds this CustomReferences field name by looking in the entity for the key that we're assigned to. */
+  get fieldName(): string {
+    return Object.entries(this.entity).filter((e) => e[1] === this)[0][0];
+  }
+
   toString(): string {
     return `CustomReference(entity: ${this.entity}, fieldName: ${this.fieldName})`;
   }
 
-  /** Finds this CustomReferences field name by looking in the entity for the key that we're assigned to. */
-  get fieldName(): string {
-    return Object.entries(this.entity).filter((e) => e[1] === this)[0][0];
+  private doGet(opts?: { withDeleted?: boolean }): U | N {
```

Comment:
Is there some reason these moved around?  Just preference to have private methods at the bottom of the file?

Reply:
Right, should have done as a separate commit, but public API / methods first, impl details hidden below.